unstable/logging/logging.go

```diff
@@ -270,26 +275,7 @@ func (w *logSinkWriter) Write(p []byte) (n int, err error) {
 }
 
 func parseTfLogEnvVar() hclog.Level {
-	env, present := os.LookupEnv(tfLogEnvVar)
-	if !present {
-		return hclog.NoLevel
-	}
-	switch strings.ToUpper(env) {
-	case "ERROR":
-		return hclog.Error
-	case "WARN":
-		return hclog.Warn
-	case "INFO":
-		return hclog.Info
-	case "TRACE":
-		return hclog.Trace
-	case "DEBUG":
-		return hclog.Debug
-	case "OFF":
-		return hclog.Off
-	default:
-		return hclog.NoLevel
-	}
+	return hclog.LevelFromString(os.Getenv(tfLogEnvVar))
```

Comment:
This is a simple refactor to use `hclog`'s own switch statement. It doesn't change any behavior.